securedrop/journalist.py

```diff
@@ -134,10 +134,14 @@ def login():
 
             if isinstance(e, LoginThrottledException):
                 login_flashed_msg += " "
-                login_flashed_msg += gettext(
+                period = seconds=Journalist._LOGIN_ATTEMPT_PERIOD
```

Comment:
Nit: spacing around the second equals sign.

Reply:
it was an accident, thanks for noticing :-)